src/towncrier/_builder.py

```diff
@@ -38,6 +38,8 @@ def parse_newfragment_basename(
     if len(parts) == 2:
         ticket, category = parts
         ticket = strip_if_integer_string(ticket)
+        if ticket.startswith("+"):
+            ticket = ""
```

Comment:
```suggestion
        if ticket.startswith("+"):
            # Tickets with names/IDs starting with a + are considered
            # orphan fragments.
            # Just use an empty ticket ID so that we can continue
            # procesing it.
            ticket = ""
```

Reply:
Code not done here, and where it was moved to is more explicit.